rst/dev-guide/cloud-backup-v2/_deconst.json

```diff
@@ -0,0 +1,3 @@
+{
+  "contentIDBase": "https://github.com/rackerlabs/docs-cloud-backup/v2/"
```

Comment:
:point_up: Note that these content ID bases are different: I've added a version suffix to them to distinguish them once we're shipping both.